examples/app/templates/batch-job.yaml

```diff
@@ -9,7 +9,8 @@ spec:
   template:
     spec:
       containers:
-      - command:
+      - args: {{- toYaml .Values.batchJob.pi.args | nindent 12 }}
```

Comment:
for `CronJob` `nindent 12` is correct, but for `Job` it should be less because it does not have `jobTemplate`, right?